spec/classes/galera_init_spec.rb

```diff
@@ -151,7 +155,7 @@
     end
   end
 
-  on_supported_os.each do |os,facts|
+  on_supported_os(:facterversion => '3.6').each do |os,facts|
```

Comment:
By default, rspec-puppet-facts will set `facterversion` to the version of the *gem* installed.  This will be so old, it doesn't have all the structured `os` facts needed for the hiera lookups to work.